lib/org/openpsa/user/accounthelper.php

```diff
@@ -140,6 +126,28 @@ protected function prepare_mail(org_openpsa_mail $mail)
         $mail->parameters["SITE_URL"] = midcom::get()->config->get('midcom_site_url');
     }
 
+    /**
+     * Send mail
+     */
+    protected function send_mail(string $username, string $usermail, string $password) : bool
```

Comment:
any particular reason why this is protected instead of private? Is there a child calls that needs this?